employees/templates/employees/report_list.html

```diff
@@ -42,7 +42,7 @@ <h1>{{ UI_text.PAGE_TITLE.value }}</h1>
                     {{ report.work_hours }}
                 </td>
                 <td>
-                    {{ report.description }}
+                    {{ report.markdown_description|safe|linebreaksbr }}
```

Comment:
Could you describe the general purpose of this improvement? Is it about displaying `html` tags in code as html? If so, just `safe` filter might do it.

Reply:
Thanks for than question because now I can see better how It works and it answer for @maciejSamerdak  question. Markdown method returns code with 'html' tags so if we write to big text there, we could see an error 'value too long for type character varying(256)'.  Without 'Safe' filter our text field doesn't work good becasue there are 'html' tags and we can see them. Last one filter help our field to find when someone put enter and but now when I tested it again I think that 'linbreaks' works nicer than 'linebreaksbr', but this is only my opinion.